lib/src/header/Header.tsx

```diff
@@ -303,34 +308,41 @@ const ResponsiveMenu = styled.div`
 const ResponsiveLogoContainer = styled.div`
   max-height: ${(props) => props.theme.logoHeight};
   width: ${(props) => props.theme.logoWidth};
-  position: absolute;
-  top: 23px;
-  left: 20px;
+  display: flex;
 `;
 
-const CloseContainer = styled.div`
+const ResponsiveIconsContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const CloseAction = styled.button`
```

Comment:
Some comments about this.

**New close action**

![image](https://user-images.githubusercontent.com/44321109/189597968-8ece3507-9b01-42f1-abb4-2440ae2c2f5b.png)

**Previous close action**

![image](https://user-images.githubusercontent.com/44321109/189598120-ce3aefac-57f5-4d8b-b830-ceaa02cc9ca7.png)

In the new version, we are changing the default styles of the action, since its size was 36x36px and now is 24x24px. We need to remain as faithful as possible to the original.

Reply:
```
const CloseAction = styled.button`
  display: flex;
  flex-wrap: wrap;
  align-content: center;
  padding: 6px;
  height: 36px;
  width: 36px;
  background-color: transparent;
  border: none;

  :focus,
  :focus-visible {
    outline: ${(props) => props.theme.hamburguerFocusColor} auto 1px;
  }
`;
```
We need the width and height attributes to be set to 36px and padding 6px (as the old close action). Display flex attributes for aligning the icon and the SVG selector is not necessary since the default size of the icon is already 24x24px.

Reply:
Flex attributes doesn't make any changes, I think the problem is in the svg. I will leave it like this until we get a new icon.